Library/Homebrew/extend/pathname.rb

```diff
@@ -189,6 +189,7 @@ def atomic_write(content)
       rescue Errno::EPERM # rubocop:disable Lint/HandleExceptions
       end
 
+      tf.close # Close the file before renaming to prevent 'Device or resource busy' error
```

Comment:
I suggest…
```ruby
# Close the file before renaming to prevent the error: Device or resource busy
tf.close
```

Reply:
When does this error occur? Is it not Linux-only?

Reply:
It affects primarily NFS, which I believe can affect macOS as well.

Reply:
Ok. Would be good to add that to the comment, too.